doc/docs/Python_Tutorials/Cylindrical_Coordinates.md

```diff
@@ -693,102 +686,84 @@ def radiated_flux_cyl(dmat: float, h: float, rpos: float, m: int) -> float:
         geometry=geometry,
     )
 
-    flux_air_z = sim.add_flux(
+    flux_air_mon = sim.add_flux(
         fcen,
         0,
         1,
         mp.FluxRegion(
             center=mp.Vector3(0.5 * L, 0, 0.5 * sz - dpml),
             size=mp.Vector3(L, 0, 0),
         ),
-    )
-
-    flux_air_r = sim.add_flux(
-        fcen,
-        0,
-        1,
         mp.FluxRegion(
             center=mp.Vector3(L, 0, 0.5 * sz - dpml - 0.5 * dair),
             size=mp.Vector3(0, 0, dair),
         ),
     )
 
     sim.run(
+        mp.dft_ldos(fcen, 0, 1),
         until_after_sources=mp.stop_when_fields_decayed(
             50.0,
-            src_cmpt,
+            mp.Er,
             src_pt,
             1e-8,
         ),
     )
 
-    flux_z = mp.get_fluxes(flux_air_z)[0]
-    flux_r = mp.get_fluxes(flux_air_r)[0]
-    flux_tot = flux_r + flux_z
+    flux_air = mp.get_fluxes(flux_air_mon)[0]
 
-    return flux_tot
+    if rpos == 0:
+        dV = np.pi / (resolution**3)
+    else:
+        dV = 2 * np.pi * rpos / (resolution**2)
 
+    # total flux from point source via LDOS
+    flux_src = -np.real(sim.ldos_Fdata[0] * np.conj(sim.ldos_Jdata[0])) * dV
 
-def radiated_flux_3d(rpos: float) -> float:
-    """Computes the radiated flux in 3d Cartesian coordinates using a
-       point-dipole source in cylindrical coordinates.
+    print(f"flux-cyl:, {rpos:.2f}, {m:3d}, {flux_src:.6f}, {flux_air:.6f}")
 
-    Args:
-       rpos: position of source in radial direction.
+    return flux_air, flux_src
 
-    Returns:
-       The radiated flux in 3d Cartesian coordinates.
-    """
+
+if __name__ == "__main__":
     layer_thickness = 0.7 * wvl / n
     dipole_height = 0.5
 
-    if rpos == 0:
-        # r = 0 source requires a single simulation with m = ±1
-        m = 1
-        flux_cyl = radiated_flux_cyl(
-            layer_thickness,
-            dipole_height,
-            rpos,
-            m,
-        )
-
-        flux_3d = flux_cyl * 2 * (resolution / np.pi) ** 2
-
-    else:
-        # r > 0 source requires Fourier-series expansion of φ
-        flux_tol = 1e-6  # relative tolerance for flux for truncating expansion
-        cutoff_M = int(2 * rpos * 2 * np.pi * fcen * n)
-        ms = range(cutoff_M + 1)
+    # r = 0 source requires a single simulation with m = ±1
+    rpos = 0
+    m = 1
+    flux_air, flux_src = led_flux(
+        layer_thickness,
+        dipole_height,
+        rpos,
+        m,
+    )
+    ext_eff = flux_air / flux_src
+    print(f"exteff:, {rpos}, {ext_eff:.6f}")
+
+    # r > 0 source requires Fourier-series expansion of φ
+    flux_thresh = 1e-5  # threshold value for flux for truncating expansion
+    rpos = [3.5, 6.7, 9.5]
+    for rp in rpos:
+        cutoff_M = int(2 * rp * 2 * np.pi * fcen * n)
```

Comment:
We don't really mention where this comes from in the tutorial description... at the very least a comment in the code would be useful.

Reply:
Added some notes in the main text to describe how this expression was derived.